API/status.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2021 Humanitarian OpenStreetmap Team
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Humanitarian OpenStreetmap Team
+# 1100 13th Street NW Suite 800 Washington, D.C. 20005
+# <info@hotosm.org>
+
+"""
+  Router Responsible for Data Source Update Status
+"""
+
+from fastapi import APIRouter
+from src.galaxy.validation.models import DataOutput, DataSource, DataRecencyParams
+from src.galaxy.app import Status
+router = APIRouter(prefix="/status")
+
+@router.post("/")
+def data_recency_status(params: DataRecencyParams):
+    """Gives the time lapse since the last update per data source per data output"""
```

Comment:
Documentation required with example request and response !

Reply:
It would be better if we can include documentation like this , with all possible combination 
https://galaxy-api.hotosm.org/v1/redoc#operation/get_mapathon_summary_mapathon_summary__post